scio-bigtable/src/main/scala/com/spotify/scio/bigtable/TableAdmin.scala

```diff
@@ -50,17 +50,41 @@ object TableAdmin {
     result
   }
 
+  /**
+   * Retrieves a set of tables from the given instancePath.
+   *
+   * @param client Client for calling Bigtable.
+   * @param instancePath String of the form "projects/$project/instances/$instance".
+   * @return
+   */
+  private def fetchTables(client: BigtableTableAdminClient, instancePath: String): Set[String] = {
+    client
+      .listTables(
+        ListTablesRequest
+          .newBuilder()
+          .setParent(instancePath)
+          .build()
+      )
+      .getTablesList
+      .asScala
+      .map(t => t.getName)
+      .toSet
+  }
+
   /**
    * Ensure that tables and column families exist.
    * Checks for existence of tables or creates them if they do not exist.  Also checks for
    * existence of column families within each table and creates them if they do not exist.
    *
    * @param tablesAndColumnFamilies A map of tables and column families.  Keys are table names.
    *                                Values are a list of column family names.
+   * @param cellExpiration The duration before which garbage collection of a cell may occur.
+   *                       Note: minimum granularity is second.
    */
   def ensureTables(
     bigtableOptions: BigtableOptions,
-    tablesAndColumnFamilies: Map[String, List[String]]
+    tablesAndColumnFamilies: Map[String, List[String]],
```

Comment:
@brianmartin would it make sense to have Duration per column family? `Map[String, List[(String, Option[Duration])]]`?

Reply:
Hm, interesting.. Definitely more flexibility there. For our immediate use case a single `cellExpiration` makes sense. I'll defer to you here.

Reply:
That's what I was thinking initially and maybe it's easier to get backward compatibility working?